docs/appendix/glossary/resource-config.md

```diff
@@ -2,8 +2,10 @@
 title: Resource Config
 id: resource-config
 full_link:
-short_description: The configuration element of either a component or a service. Typically expressed in JSON.
+short_description: The configuration element of either a component or a service.
```

Comment:
```suggestion
short_description: The configuration element of a component, service or other resource.
```
I'm actually not 100% sure whether we want "resource" to include modules and remotes, or if those indirectly fall within components and services anyway (since modules are components or services, and remotes provide access to components and services of other robots). So this suggestion and the one below should be ignored if we just want to keep it to components and services.